src/EepromInternalStorage.cpp

```diff
@@ -75,10 +75,15 @@ void EepromInternalStorage::write(unsigned int eeaddress, byte data)
 //
 void EepromInternalStorage::writeBytes(unsigned int eeaddress, const byte src[], byte numbytes)
 {
+  #ifdef __SAM3X8E__
+  // Shouldn't use this for the Due.
+  #else
   for (byte i = 0; i < numbytes; i++)
   {
-    setChipEEPROMVal(eeaddress + i, src[i]);
+    EEPROM.write(eeaddress + i, src[i]);
```

Comment:
You can change this back to use setChipEEPROMVal() now that it doesn't commit. Then you can remove the #ifdef for __SAM3X8E__. It is enough to have that #ifdef in the low level functions.